src/definitions/modules/sidebar.js

```diff
@@ -309,7 +309,7 @@
                     module.verbose('Forcing repaint event');
                     element.style.display = 'none';
                     var ignored = element.offsetHeight;
-                    element.scrollTop = element.scrollTop;
+                    element.scrollTop = element.scrollTop; // eslint-disable-line no-self-assign
```

Comment:
is this line really useful? can we unify the (forced) repaint across modules?

Reply:
> is this line really useful?

From the git history, is is not clear, why this line was necessary (clear commit message ftw). I strongly believe it was needed by IE8-11 (?) back then.

>  can we unify the (forced) repaint across modules?

Otherwise, there are a lot of functions i would remove from all modules and put into a separate "core.js" or similar  (like the debug functions). maybe enhance the existing site.js. 

Perhaps repaint is also a candidate.

But this would mean each module is not working on its own anymore (at least when following the ES5 syntax as it it now).
I am already thinking about such way to reduce the code overhead. Even if that means people have to make sure to include core.js...but this is another story